ui/app/js/controllers/release_new_controller.js

```diff
@@ -18,6 +18,28 @@ function($scope, $http, $modalInstance, CSRF, Releases, releases) {
   $scope.errors = {};
   $scope.saving = false;
 
+  $scope.fillName = function () {
+    var file = $scope.dataFile;
+
+    var reader = new FileReader();
+    reader.onloadend = function(evt) {
+      var blob = evt.target.result;
+      $scope.$apply( function() {
+         $scope.release.name = JSON.parse(blob).name;
```

Comment:
It is, but when a user selects the JSON file locally, there's no guarantee that they've remembered to include "name" in it. The best thing to do here is probably set an error for this field.